articles/connections/passwordless/configure-passwordless-connection.md

```diff
@@ -0,0 +1,137 @@
+---
+title: Configure Passwordless Connections
+topics:
+    - connections
+    - passwordless
+    - sms
+    - email
+contentType: how-to
+useCase: customize-connections
+---
+# Configure Passwordless Connections
+
+This guide will show you how to configure Passwordless connections using Auth0's Dashboard. This task can also be performed using the Management API.
+
+## Configure Passwordless using SMS messages
+
+::: panel Twilio Account
+Before proceeding, you will need a valid Twilio account. Auth0 will use your [Twilio Account SID](https://www.twilio.com/help/faq/twilio-basics/what-is-an-application-sid) and [ Auth Token](https://www.twilio.com/help/faq/twilio-basics/what-is-the-auth-token-and-how-can-i-change-it) to send SMS messages tousers.
+:::
+
+1. Navigate to the [Connections > Passwordless](${manage_url}/#/connections/passwordless) page in the [Auth0 Dashboard](${manage_url}/), and enable the SMS toggle.
+
+![Enable SMS Passwordless](/media/articles/connections/passwordless/connections-passwordless-list.png)
+
+2. Enter your **Twilio Account SID** and **Twilio Auth Token**.
+
+::: note
+To learn how to find your Twilio SID and Auth Token, see the Twilio docs on the[Application SID](https://support.twilio.com/hc/en-us/articles/223136607-What-is-an-Application-SID-) and the [AuthToken](https://support.twilio.com/hc/en-us/articles/223136027-Auth-Tokens-and-How-to-Change-Them).
+:::
+
+![Configure SMS Passwordless](/media/articles/connections/passwordless/connections-passwordless-sms.png)
+
+3. Select your **SMS Source** and, depending on your selection, enter either your **Twilio Messaging Service SID** or a **From** phone number. Users will see what you enter as the sender of the SMS.
+
+::: note
+To learn about using Twilio Copilot, see [Sending Messages with Copilot](https://www.twilio.com/docs/api/rest/sending-messages-copilot).
+:::
+
+4. In **Message**, enter the body text of the SMS.
+
+**Make sure that your Message includes `@@password@@` since this is the placeholder that will be replaced with the one-time password that is sent to the user.**
+
+You can use multiple languages in your SMS Messages. You can choose the language used by calling the [`passwordless/start`](/docs/api/authentication/reference#get-code-or-link) endpoint and setting the `x-request-language` header to the language of your choice. Otherwise, the language used will be extracted from the `accept-language` header (which is automatically set by your browser).
+
+The **Message** area accepts Markdown and Liquid syntax. If you choose to use Liquid, you can programmatically construct elements of the message. For example, you can reference the `request_language` parameter to change the language of the message:
```

Comment:
Link to this doc about Liquid syntax?
https://auth0.com/docs/email/liquid-syntax